app/src/main/res/values/strings.xml

```diff
@@ -36,35 +36,35 @@
     <string name="showOnlyApkBackedUp">Apps with only APK backed up</string>
     <string name="showOnlyDataBackedUp">Apps with only data backed up</string>
     <string name="sortByLabel">Sort by label</string>
-    <string name="sortByPackageName">Sort by packagename</string>
+    <string name="sortByPackageName">Sort by package name</string>
     <string name="sort">Sort and filter</string>
     <string name="uninstallDialogMessage">Uninstall this app?</string>
     <string name="deleteBackupDialogMessage">Delete backup files?</string>
-    <string name="restoreDataWithoutApkError">Cannot restore data without restoring APK, package is not installed</string>
+    <string name="restoreDataWithoutApkError">Cannot restore data without restoring APK. The package was not installed.</string>
     <string name="collectingData">Collecting package info…</string>
-    <string name="utilboxCheck" translatable="false">Checking for busybox/toybox…</string>
+    <string name="utilboxCheck" translatable="false">Checking for BusyBox/Toybox…</string>
     <string name="noSu">Couldn\'t get superuser permissions</string>
-    <string name="busyboxProblem">Couldn\'t find busybox. please check error log</string>
+    <string name="busyboxProblem">Could not find BusyBox. Please check the error log.</string>
     <!-- mkfileError: an error occured while creating a file. format: string + " " + filename -->
     <string name="mkfileError">Couldn\'t create</string>
     <!-- fallbackToDefault: indicates a fallback to the default for creation of a specific file. used in the same way as mkfileError -->
     <string name="fallbackToDefault">Fall back to default</string>
-    <string name="backupFolderError">Please set the correct path in preferences and ensure it exists or is createable</string>
```

Comment:
This is a reference error. Like that it should be in \"Preferences\". but that is a bit quirky still.